software/NRG_itho_wifi/10_Gen_func.ino

```diff
@@ -553,22 +553,26 @@ void logInput(const char * inputString) {
 
 void execAutoPilot()
 {
+  logInput("AUTOPILOT: inside");
```

Comment:
If called from the TaskMQTT loop this would be called 40x a sec and starts a write to flash, not a good idea.

Reply:
Never realized that but never dug into logInput(). thought it was using a buffer and just dumped the buffer to a file when clicked download file.